Formula/arangodb.rb

```diff
@@ -23,41 +26,91 @@ class Arangodb < Formula
   end
 
   def install
-    # clang on 10.8 will still try to build against libstdc++,
-    # which fails because it doesn't have the C++0x features
-    # arangodb requires.
     ENV.libcxx
 
-    args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --disable-relative
-      --datadir=#{share}
-      --localstatedir=#{var}
-    ]
+    if build.head?
+      args = %W[
```

Comment:
sound good, will add std_cmake_args. However, USE_OPTIMIZE_FOR_ARCHITECTURE, ASM_OPTIMIZATIONS, ETCDIR, VARDIR are specific to ArangoDB.